gui/dialogs/aircraft-dialog.xml

```diff
@@ -113,15 +113,6 @@
                 <binding>
                     <command>dialog-apply</command>
                 </binding>
-                <binding>
```

Comment:
Why did you remove this?

Reply:
That's for the repair button, this is for "Damage" checkbox. If you take a look at the changes here on Github, you'll see red lines; code that has been removed. Github doesn't make things up :wink:

Reply:
@onox, I really can't remember why I may have removed that code in this PR. However, after testing, this is the correct behavior. I know we were not happy with the way the "Damage" check box worked. Because, when you were flying along and you unchecked the check box, the engine and everything would be reset. We didn't like that. If the aircraft is not damaged and you decide to un-check the box and not allow damage to occur anymore, nothing should happen in the sim. You should keep on flying and simply not allow damage anymore. But we also wanted the un-checking of the "Damage" check box to repair the aircraft it it was damaged at the time you un-check it so as not to leave the aircraft in a damaged state with no option to repair because you turned off "Damage". I probably figured out this needed to be removed to get everything working correctly (which it does) and decided to do it in this PR instead of creating another PR. I must have forgotten to note it in the commit message and for that I apologize. The damage code and logic in the menu and this PR is correct. SO GET OFF MY BACK! :rofl: Just kidding!
Again I am really sorry for how disjointed this looks and that I don't have a precise reason for doing it here. I'm sure there was a reason I did it here and at this time. It's just been too long since I worked on it to remember exactly why here and not in a more appropriate PR. More than likely it was because I had already closed out the appropriate PR and we figured out this still needed to be done. @gilbertohasnofb  you were involved in the discussions that lead to the final outcome of the damage system menu logic, maybe you can test this to confirm that it is exactly the behavior we wanted. I just tested it and works as I described above which is what I remember we wanted. This change also went hand in hand with https://github.com/c172p-team/c172p-detailed/pull/1053